### DISCUSSION TITLE
Understand artifact-download

### DISCUSSION THREAD

**Discussion by KodaCHC** [community/community#25277] [Actions]
<p>I am trying to understand the action artifact-download. Unfortunately I have a mess here and hope you can help me.</p>
<p>What exactly does artifact-download do? Actually, I thought that this step is necessary (i have read the repository README) so that a ZIP file is offered to me at the end of my workflow. If I delete artifact-download, however, I am still offered a ZIP file.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The artifact file you can download after the workflow is created using <a href="https://github.com/actions/upload-artifact">actions/upload-artifact</a>. If that’s all you want, you don’t need actions/download-artifact.</p>
<p><a href="https://github.com/actions/download-artifact">actions/download-artifact</a> is useful if you want to use that artifact in later parts of your workflow (especially other jobs).</p>
